internal/pubsub/pubsub.go

```diff
@@ -69,13 +69,6 @@ func New(self wrp.DeviceID, opts ...Option) (*PubSub, error) {
 		),
 	}
 
-	defaults := []Option{
-		WithPublishTimeout(5 * time.Second),
-	}
-
-	// Prepend the defaults to the provided options.
-	opts = append(defaults, opts...)
-
```

Comment:
removing in order to keep things consistent (where our defaults are set), since we already set defaults in `cmd/xmidt-agent/config.go`